src/nft/components/collection/CollectionStats.tsx

```diff
@@ -353,6 +354,10 @@ const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMob
   const floorChangeStr = Math.round(Math.abs(stats?.stats?.one_day_floor_change ?? 0))
   const arrow = stats?.stats?.one_day_floor_change ? getDeltaArrow(stats.stats.one_day_floor_change) : undefined
 
+  const isBagExpanded = useBag((state) => state.bagExpanded)
+  const isScreenSize = useScreenSize()
+  const isSmallContainer = isMobile || (!isScreenSize['xl'] && isBagExpanded)
```

Comment:
I think we can lower the screen size req, missing a lot of potential stats at this width
<img width="608" alt="Screen Shot 2022-12-13 at 15 50 51 " src="https://user-images.githubusercontent.com/11512321/207462155-3933131c-bd00-435b-831b-b285cd316c2a.png">